src/less/carousel/stories/carousel.stories.js

```diff
@@ -131,8 +131,8 @@ export const RTL = () => `
                 <span>Top Products - Slide 1 of 4</span>
             </h4>
             <button class="carousel__control carousel__control--prev" aria-label="Previous Slide - Top Products">
-                <svg aria-hidden="true" class="icon icon--carousel-next" focusable="false">
-                    <use xlink:href="#icon-carousel-next"></use>
+                <svg aria-hidden="true" class="icon icon--carousel-prev" focusable="false">
+                    <use xlink:href="#icon-carousel-prev"></use>
```

Comment:
this was a correction - I had previously switched these, but that switch was in error.